package.json

```diff
@@ -5,7 +5,7 @@
   "main": "app.js",
   "scripts": {
     "start": "node src/app.js",
-    "test": "jest",
+    "test": "jest --env=node",
```

Comment:
This part doesn't require review, just help me comment under what the --env=node does make I dey follow along

Reply:
I have removed the --env=node. It is not necessary.